pytest_allure_dsl.py

```diff
@@ -198,6 +204,16 @@ def _yaml_load(self, string):
             self.description_load_error = e
             return {}
 
+    def _check_steps_was_used(self):
```

Comment:
may be shorter:

```python
def _check_steps_was_used(self):
    if self._steps_was_used < self.steps.keys():
        raise StepsWasNotUsed(
                 *list(set(self.steps.keys()) - set(self._steps_was_used))
            )
```

Reply:
How we know about not implemented steps? What we must delete from steps? pdb? breakpoint?

Reply:
Oh, sorry, i think len(...) - len(...) :) I can change it if you wish, but this is overhead

Reply:
Not implemented steps will be passed to exception, if any

Reply:
As you wish :)

Reply:
@griddic ?

Reply:
upd: more clean version, see edited first comment

Reply:
@igorozt It's not working for python2, but supporting for this lib, TypeError will raised

Reply:
will work both for python 2 and 3:

`not_used_steps = set(self.steps.keys()) - set(self._steps_was_used)`

but consider https://pythonclock.org :)

Reply:
Yep! )

Reply:
> @griddic ?

I still prefer _steps_was_used to be a set )

Reply:
@griddic @igorozt I fixed as you wish :)

Reply:
Looks nice, except for tuple(..) which seems unnecessary